tests/test_flows/test_sklearn.py

```diff
@@ -304,15 +350,17 @@ def test_serialize_pipeline_clustering(self):
         new_model.fit(self.X, self.y)
 
     def test_serialize_feature_union(self):
-        ohe = sklearn.preprocessing.OneHotEncoder(sparse=False)
+        ohe_params = {'sparse': False}
+        if LooseVersion(sklearn.__version__) >= "0.20":
+            ohe_params['categories'] = 'auto'
+        ohe = sklearn.preprocessing.OneHotEncoder(**ohe_params)
         scaler = sklearn.preprocessing.StandardScaler()
-        fu = sklearn.pipeline.FeatureUnion(transformer_list=[('ohe', ohe),
-                                                             ('scaler', scaler)])
-        serialization =  sklearn_to_flow(fu)
-        self.assertEqual(serialization.name,
-                         'sklearn.pipeline.FeatureUnion('
-                         'ohe=sklearn.preprocessing.data.OneHotEncoder,'
-                         'scaler=sklearn.preprocessing.data.StandardScaler)')
+        fu = sklearn.pipeline.FeatureUnion(
+            transformer_list=[('ohe', ohe), ('scaler', scaler)])
+        serialization = sklearn_to_flow(fu)
+        estimators_names = ('FeatureUnion', 'OneHotEncoder', 'StandardScaler')
+        for keyword in estimators_names:
```

Comment:
What's the reason to make this test more leniant?

Reply:
We moved some of the transformer from module. Therefore, the serialization name will be incorrect across version of scikit-learn. In the future, it could be as well the case.

I have 2 rermarks:

* The test was intended to check the presence of the estimator in the serialization, doesn't it? Is it something else in the test which was important to be tested?
* Speaking with @janvanrijn IRL, currently all scikit-learn estimators are imported with only a single level and no more. All extra level are actually kinda private (some are explicit by putting some "_" at the front of the module name). I think that you should not rely on those extra levels.

To be more specific, you should rely that a user do `from sklearn.decompisition import PCA` and not `from sklearn.decomposition.pca import PCA` since `pca` should be private module.

Reply:
* The test also intends to check that the name is correctly serialized and does not change (also due to mistakes on our side). 
* Regarding the names: they extra module level might be private, but that's the information we get from the Python interpreter when we ask it for the full module path. Do you have any other suggestion on how to obtain the name(s)?

Reply:
I think the idea of having such a mapping is a good idea from the point of view of the OpenML connector, but it needs to live inside of scikit-learn in order to get constant maintenance. For now I suggest to open a new issue in the OpenML-python issue tracker and the scikit-learn issue tracker to keep track of this and get this PR merged. Would you be okay to re-add the string asserts and add to add an if/else statement?